Basic/DyanmicPrefabs/Assets/Scripts/01_Sparse loading with no latejoin/SparseLoadingNoLatejoinSample.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Threading.Tasks;
+using Unity.Collections;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using UnityEngine.UI;
+
+namespace Game
+{
+    /// <summary>
+    /// This is probably the most advanced scenario - we are spawning prefabs that aren't known to clients beforehand.
+    ///
+    /// Here we are not implementing client acknowledgement of the prefab loading completion -
+    /// this would allow the server to wait for all the clients to have loaded the prefab and spawn it only then, which is safer.
+    /// However, this is not implemented here for simplicity - there is a whole slew of potential failures that this would have to handle.
```

Comment:
Our goal is to show patterns to help users with their production games. Shouldn't we show the kind of errors they'd need to handle?
Or are those the errors we were talking about that feel like bugs? If so I'd reference the jira ticket that's waiting for this

Reply:
Well, this is probably poor phrasing on my part. What I meant is:
 - you can implement a thing that waits for all clients to report to you that something has loaded, so that the server knows for a fact that all clients have loaded the new prefab that it wants to spawn
 - however
   - presently there is no way to deal with failed spawn (imagine we failed to load the asset in time, due to the fact that it's streamed from CDN (this is a crazy scenario imo, but it's technically possible to do), in fact the message will get discarded (see DeferredMessagingTests.cs WhenAMessageIsDeferredForMoreThanTheConfiguredTime_ItIsRemoved )
   - so the usefulness of this more complex spawning logic is presently equal to zero, if we wanted to be able to actually recover from failed loads
   
   Chatting with the SDK folks about the ability to recover from failed spawn.

Reply:
Changed the implementation to have the safety mechanism via client acknowledgement and server being able to cancel the spawn if clients fail to load the prefabs.